src/devices/Ssd1306/README.md

```diff
@@ -0,0 +1,18 @@
+﻿# Solomon Systech Ssd1306
+
+## Summary
```

Comment:
I was just thinking that one thing we should probably try to add to all these READMEs in the future is Connection Type. That way, consumers know if they need to enable I2c, SPI or PWM and also know right of the bat if their board is compatible with the binding. Nothing really to address in your PR, just food for thought :)

Reply:
That's a good idea.  It seems like we could have a list of all that is possible to use even if the binding doesn't support yet.  For example, this binding will eventually support SPI.  So we could have a list of I2C and SPI with I2C checked for now.

Where would this section be?  Top right under title?  First thing under Binding Notes? Other?

Reply:
Agreed, having the checkboxes of supported protocols make a lot of sense, I already know we have a few temperature sensors that support SPI but our bindings don't so we should add that. I don't have a preference on where to have this information on the Readme since they are usually not too large, but yeah having it either before or after binding notes sounds good to me.